embulk-output-sqlserver/src/main/java/org/embulk/output/SQLServerOutputPlugin.java

```diff
@@ -87,12 +111,45 @@ protected Features getFeatures(PluginTask task)
     protected SQLServerOutputConnector getConnector(PluginTask task, boolean retryableMetadataOperation)
     {
         SQLServerPluginTask sqlServerTask = (SQLServerPluginTask) task;
+        Driver driver;
+        boolean useJtdsDriver = false;
 
         if (sqlServerTask.getDriverPath().isPresent()) {
             loadDriverJar(sqlServerTask.getDriverPath().get());
+            try {
+                driver = (Driver) Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver").newInstance();
+            } catch(Exception e) {
+                throw new ConfigException("Driver set at field 'driver_path' doesn't include Microsoft SQLServerDriver", e);
+            }
+        } else {
+            // prefer Microsoft SQLServerDriver if it is in classpath
+            try {
+                driver = (Driver) Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver").newInstance();
+            } catch(Exception e) {
+                logger.info("Using jTDS Driver");
+                driver = new net.sourceforge.jtds.jdbc.Driver();
+                useJtdsDriver = true;
+            }
         }
 
+        UrlAndProperties urlProps = getUrlAndProperties(sqlServerTask, useJtdsDriver);
+        logger.info("Connecting to {} options {}", urlProps.getUrl(), urlProps.getProps());
```

Comment:
We shouldn't log database password for security reason.
(Original `SQLServerOutputPlugin` logs properties before setting password.)